master.yaml

```diff
@@ -166,6 +167,37 @@ Resources:
                 ProjSettingsDir: transportationAPI
                 Path: /transport*
 
+    Civic2017Service:
+        Type: AWS::CloudFormation::Stack
+        Properties:
+            TemplateURL: https://s3-us-west-2.amazonaws.com/hacko-infrastructure-cfn/services/civic-2017-service/service.yaml
+            Parameters:
+                VPC: !GetAtt VPC.Outputs.VPC
+                Cluster: !GetAtt ECS.Outputs.Cluster
+                DesiredCount: 2
+                Listener: !GetAtt ALB.Outputs.Listener
+                ConfigBucket: hacko-civic2017-config
+                DeployTarget: integration
+                ProjSettingsDir: civic-2017_proj  
+                Path: /2017*
+
+    Civic2018Service:
+       Type: AWS::CloudFormation::Stack
+       Properties:
+           TemplateURL: https://s3-us-west-2.amazonaws.com/hacko-infrastructure-cfn/services/civic-2018-service/service.yaml
+           Parameters:
+               VPC: !GetAtt VPC.Outputs.VPC
+               Cluster: !GetAtt ECS.Outputs.Cluster
+               DesiredCount: 2
+               Listener: !GetAtt ALB.Outputs.Listener
+               ConfigBucket: hacko-civic2018-config
+               DeployTarget: integration
+               ProjSettingsDir: civic-2018_proj  
+               Path: /2018*
+
```

Comment:
```
               ConfigBucket: hacko-civic2018-config
               DeployTarget: integration
               ProjSettingsDir: civic-2018_proj  
```
Should this be changed to no environment variables at all, based on @DingoEatingFuzz's comment?

Reply:
I left them in for cosistancy from the previous services, since the new services do not use them ( at least right now we believe the new service build/deploy does not use them, having them defined is a noop (ie. They exist but no one uses them) they can be removed at a later time or if folks want them removed, I can create a new issue, branch and remove them now.